src/components/MasterItems.js

```diff
@@ -62,6 +75,39 @@ class MasterItems extends Component {
           'Add Item'
         )
       ),
+      dom.div(
```

Comment:
This is just moved from `NewOrder` component right? Seems like we just kinda moved everything over haha..